src/main/java/org/openstreetmap/atlas/checks/validation/tag/InvalidAccessTagCheck.java

```diff
@@ -121,6 +106,22 @@ private boolean hasBoundaryNode(final AtlasObject object)
         return Optional.empty();
     }
 
+    /**
+     * Checks if {@link AtlasObject} contains synthetic boundary Node
+     *
+     * @param object
+     *            the {@link AtlasObject} to check
+     * @return true if edge contains synthetic boundary Node.
+     */
+    private boolean hasBoundaryNode(final AtlasObject object)
+    {
+        if (object instanceof Edge && ((Edge) object).connectedNodes().stream().anyMatch(SyntheticBoundaryNodeTag::isBoundaryNode))
```

Comment:
this his be changed to a return (and get rid of the rest of the function)?
```suggestion
        return (object instanceof Edge && ((Edge) object).connectedNodes().stream().anyMatch(SyntheticBoundaryNodeTag::isBoundaryNode))
```

Reply:
Fixed! Thanks!